mon-pix/app/components/signin-form.js

```diff
@@ -17,17 +23,35 @@ export default class SigninForm extends Component {
   @action
   async signin(event) {
     event && event.preventDefault();
-    
     this.hasFailed = false;
     try {
       await this.args.authenticateUser(this.username, this.password);
-    } catch (err) {
-      const title = ('errors' in err) ? err.errors.get('firstObject').title : null;
-
-      if (title === 'PasswordShouldChange') {
+    } catch (response) {
+      const firstError = _.get(response, 'errors[0]');
+      if (_.get(firstError, 'title') === 'PasswordShouldChange') {
```

Comment:
```suggestion
      const passwordExpiredError = (_.get(firstError, 'title') === 'PasswordShouldChange');
      if (passwordExpiredError) {
```

Reply:
Done en rajoutant un petit is pour le boolean